### ISSUE TITLE
Text edit to section "Why do we include the privacy notice" on preview comment page

### ISSUE THREAD

**Issue by katska** [openaustralia/planningalerts#1866]
On a page such as https://www.planningalerts.org.au/comments/166670/preview

![Screenshot 2024-07-26 at 3 35 34 PM](https://github.com/user-attachments/assets/12f2c904-6255-49b8-9b62-f712b61a0c36)

Using "force" twice in close proximity seems a little clunky when I scanned it today, could the second use of the word "force" be replaced with "make"? Or if there's a desire to emphasise we're **forcing** them, then please say so, it's a different strategy and I'm happy to think of changes to make that really sing! Noting I like the words "assert" and "affirm" relating to standing up for / advocating the commenter's right for their personal information to be kept safe. (in case I come back to this and we like sending the language that way). 

Why do we include the privacy notice?

Different planning authorities handle your private information (email address and street address) in all kinds of different ways.

This privacy notice is our attempt to **force** planning authorities to give you the same privacy protection as provided by Planning Alerts and **force** them to ask for your active consent if they want to publish or pass on your private information.

**Comment by JoannaHill** [comment]
<img width="780" alt="Screen Shot 2024-07-29 at 4 53 19 pm" src="https://github.com/user-attachments/assets/20e22a4d-ddae-470d-a6fd-bcea370e84ba">
Updating to correct screengrab

**Comment by JoannaHill** [comment]
I agree the use of the word 'force' twice in the same sentence may be a little strong. @katska be cool if you make a replacement sentence suggestion here.
